### PR TITLE
Document that middleware factories are run for every request.

### PR DESCRIPTION
**Pull request by butla**
I guess the title says it all. Related to #2225

### CODE REVIEW

**Comment by butla** [comment]
Wait a sec, I should have based that on the released tag... Gotta change the branch.

File: docs/web.rst [hunk 1]
```diff
@@ -904,6 +904,10 @@ the **next** *middleware factory*. The last *middleware factory* always receives
 the :ref:`request handler <aiohttp-web-handler>` selected by the router itself
 (by :meth:`UrlDispatcher.resolve`).
 
+.. note::
+
+   The chain of factories is run for every handled request.
```
Comment (samuelcolvin) [inline docs/web.rst]: I think someone could still read this and think it's just reiterating that each middleware is called for all views.

Perhaps we could be more explicit, something like 

"Both the outer `middleware_factory` coroutine and the inner `middleware_handler` coroutine are called for every request handled."

**Comment by butla** [comment]
@samuelcolvin Good idea, changed according to your suggestion.

**Comment by samuelcolvin** [comment]
done. Thanks @butla.

@asvetlov depending on the outcome of #2225 we might need to add this to master too, do we have somewhere to record that so we don't forget?

**Comment by asvetlov** [comment]
I've merged 2.2 into master.

**Comment by asvetlov** [comment]
@samuelcolvin in aiohttp we do merging release branches like `2.2` into master after every PR like this.
Well, the rule was never pronounced but I always did it.
aiohttp committers, please do it too.
After switching changenotes to towncrier it is not painful job, we have no merge conflicts in CHANGES.rst anymore.

**Comment by samuelcolvin** [comment]
humm, I'm not seeing this commit or the change on master?

**Comment by asvetlov** [comment]
Missed to make `git push` before sending the message.

**Pull request merged**